### ISSUE TITLE
Undefined variable 'print_obj_idc'

### ISSUE THREAD

**Issue by Druidefix** [Fiech/surf3tikz#2]
In the current version the following results in an error:

```
surf(randn(10), 'EdgeColor','None')
view(0,90)
surf3tikz(gcf, 'schlendi', cfg)
```

For the case of an 2D plot the variable 'print_obj_idc' is not defined.

**Comment by Fiech** [comment]
Ok, so, even without a cfg (whose content I can't assume), this is a very stupid bug stemming from my latest simplification.

Funnily enough, the creation of a surface plot by only providing the ZData revealed a separate bug, described in issue #3.

I will soon provide fix commit for both issues in one push.

**Comment by Fiech** [comment]
Fixed with commit ba7d9e9
